packages/hash/frontend/src/assets/svg/hash-nav.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="100" height="18" viewBox="0 0 100 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M25.0742 0.537109H29.738V6.87204H35.7239V0.537109H40.3877V17.4625H35.7239V11.0308H29.738V17.4625H25.0742V0.537109Z" fill="#4D5C6C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M55.6059 0.537109H51.1013L43.9229 17.5197H48.837L50.0414 14.5086H56.5454L57.7739 17.5197H62.7844L55.6059 0.537109ZM53.3175 6.07753L55.1963 10.8712H51.4145L53.3175 6.07753Z" fill="#4D5C6C"/>
+<path d="M72.9468 17.4625C71.4313 17.4625 69.9924 17.2365 68.6301 16.7845C67.2677 16.3326 66.0626 15.6546 65.0146 14.7507L67.6265 11.735C69.3193 13.0441 71.1492 13.6987 73.1161 13.6987C73.7449 13.6987 74.2245 13.6013 74.555 13.4065C74.8855 13.2117 75.0508 12.935 75.0508 12.5766V12.5298C75.0508 12.3584 75.0105 12.2064 74.9298 12.0739C74.8492 11.9415 74.7001 11.8129 74.4825 11.6882C74.2648 11.5635 73.9746 11.4428 73.6118 11.3259C73.2491 11.209 72.7775 11.0882 72.1971 10.9635C71.2943 10.7609 70.4559 10.5388 69.682 10.2973C68.9082 10.0557 68.2351 9.74399 67.6627 9.36215C67.0904 8.98031 66.639 8.50497 66.3085 7.93612C65.978 7.36726 65.8127 6.66204 65.8127 5.82044V5.77369C65.8127 5.01002 65.9618 4.3087 66.2601 3.66971C66.5584 3.03072 66.9937 2.47745 67.566 2.0099C68.1383 1.54234 68.8276 1.17999 69.6337 0.92284C70.4398 0.665685 71.3507 0.537109 72.3664 0.537109C73.8174 0.537109 75.1112 0.720232 76.2478 1.08648C77.3845 1.45273 78.4123 1.99431 79.3312 2.71123L76.9854 5.91395C76.2116 5.38406 75.4175 4.98275 74.6034 4.71001C73.7892 4.43727 73.0032 4.3009 72.2455 4.3009C71.6812 4.3009 71.258 4.4022 70.9759 4.60481C70.6937 4.80742 70.5526 5.05678 70.5526 5.35289V5.39965C70.5526 5.58667 70.597 5.75031 70.6857 5.89058C70.7743 6.03084 70.9275 6.15942 71.1451 6.27631C71.3628 6.3932 71.6611 6.51008 72.0399 6.62697C72.4188 6.74386 72.8984 6.86464 73.4788 6.98932C74.4462 7.19193 75.3208 7.4296 76.1027 7.70234C76.8847 7.97508 77.5497 8.30626 78.0979 8.69589C78.646 9.08552 79.0652 9.55696 79.3554 10.1102C79.6456 10.6635 79.7907 11.322 79.7907 12.0856V12.1324C79.7907 12.974 79.6255 13.7221 79.295 14.3766C78.9644 15.0312 78.5009 15.5884 77.9044 16.0481C77.3079 16.5079 76.5904 16.8586 75.7521 17.1001C74.9137 17.3417 73.9786 17.4625 72.9468 17.4625Z" fill="#4D5C6C"/>
+<path d="M83.9102 0.537109H88.5739V6.87204H94.5599V0.537109H99.2236V17.4625H94.5599V11.0308H88.5739V17.4625H83.9102V0.537109Z" fill="#4D5C6C"/>
+<path opacity="0.886277" d="M10.0918 18H15.1036V0H10.0918V18Z" fill="#4D5C6C"/>
+<path opacity="0.898345" d="M2.89648 18H7.90794V0H2.89648V18Z" fill="#4D5C6C"/>
+<path opacity="0.881278" d="M0 7.71497H18V2.70312H0V7.71497Z" fill="#4D5C6C"/>
+<path opacity="0.855632" d="M0 15.0398H18V10.0283H0V15.0398Z" fill="#4D5C6C"/>
+</svg>
```

Comment:
I think this should be wrapped in the `SvgIcon` MUI component, and defined/exported as a react component. If you're unsure what I mean by this, have a look at [how we are defining custom icons in the BP repo](https://github.com/blockprotocol/blockprotocol/tree/main/site/src/components/icons)

Reply:
This is done now